sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateMutableProjection.scala

```diff
@@ -124,6 +124,7 @@ object GenerateMutableProjection extends CodeGenerator[Seq[Expression], MutableP
 
         public java.lang.Object apply(java.lang.Object _i) {
           InternalRow ${ctx.INPUT_ROW} = (InternalRow) _i;
+          /*** project list: ${expressions.map(_.toCommentSafeString).mkString(", ")} */
```

Comment:
When there are many columns, the comments are far away from their code, it's better to interweave them.